pkg/network/dhcpconfigurators/bridge.go

```diff
@@ -0,0 +1,47 @@
+package dhcpconfigurators
+
+import (
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/pkg/network/cache"
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+)
+
+type BridgeConfigurator struct {
+	advertisingIfaceName string
+	cacheFactory         cache.InterfaceCacheFactory
+	handler              netdriver.NetworkHandler
+	launcherPID          string
+}
+
+func NewBridgeConfigurator(cacheFactory cache.InterfaceCacheFactory, launcherPID string, advertisingIfaceName string, handler netdriver.NetworkHandler) *BridgeConfigurator {
+	return &BridgeConfigurator{
+		advertisingIfaceName: advertisingIfaceName,
+		cacheFactory:         cacheFactory,
+		launcherPID:          launcherPID,
+		handler:              handler,
+	}
+}
+
+func (d *BridgeConfigurator) GetConfiguration(_ *v1.Interface, _ *v1.Network, podInterfaceName string) (*cache.DHCPConfig, error) {
+	return d.importConfiguration(podInterfaceName)
+}
+
+func (d *BridgeConfigurator) importConfiguration(ifaceName string) (*cache.DHCPConfig, error) {
+	return importConfiguration(d.cacheFactory, d.launcherPID, ifaceName)
+}
+
+func (d *BridgeConfigurator) ExportNonRecoverableConfiguration(config *cache.DHCPConfig) error {
+	return exportConfiguration(d.cacheFactory, d.launcherPID, *config)
+}
+
+func (d *BridgeConfigurator) EnsureDHCPServerStarted(podInterfaceName string, dhcpConfig cache.DHCPConfig, dhcpOptions *v1.DHCPOptions) error {
+	return ensureDHCPServerStarted(podInterfaceName, dhcpConfig, dhcpOptions, d.handler, d.advertisingIfaceName, d.shouldFilterByMac())
+}
+
+func (d *BridgeConfigurator) getDHCPStartedFilePath(podInterfaceName string) string {
+	return getDHCPStartedFilePath(podInterfaceName)
+}
```

Comment:
It seems to me you're attempting to bringing back something wrong: an interface, with N implementation that are exactly the same. 

This is a clean sign the encapsulation is off.